charts/couchdb/templates/deployment.yaml

```diff
@@ -32,3 +32,10 @@ spec:
           value: {{ .Values.global.couchdbPassword | default "some_passw0rd" | quote }}
         - name: "DB_PORT"
           value: {{ include "couchdb_port" . | quote }}
+        volumeMounts:
```

Comment:
It's better to check the `persistence.enabled` option here as well. If not specified, do not need this section